src/Config.js

```diff
@@ -220,6 +224,14 @@ export class Config {
     }
   }
 
+  static validateMaxLimit(maxLimit) {
+    if (maxLimit) {
```

Comment:
Can you add a test with server configured with maxLimit: 0? This will probably not throw as not goin in.